src/envConfig.ts

```diff
@@ -225,7 +225,6 @@ export const asEnvConfig = asObject({
   DEBUG_ACCOUNTBASED: asOptional(asBoolean, false),
   DEBUG_VERBOSE_ERRORS: asOptional(asBoolean, false),
   DEBUG_THEME: asOptional(asBoolean, false),
-  DISABLE_WARNINGS: asOptional(asBoolean, false),
```

Comment:
Are all the warnings gone? If not, it would be convenient to have a way to suppress them during development, but not by a persistent config in the `env.json`. Rather a flag for `yarn start` would be session based/ephemeral so devs don't forget to unset it. Example, `yarn start --mellow-yellow`.